spec/spec_helper.cr

```diff
@@ -25,13 +25,15 @@ macro assert_unauthorized(response)
   fail "expected error response not found" unless found
 end
 
-# Disable all authentication checks by making `DppmRestApi::Actions.has_access?`
-# always return true, then set the authentication back to the default system
-# after yielding to the block.
-def without_authentication!
-  DppmRestApi::Actions.access_filter = ->(_context, _permissions) { true }
-  yield
-  DppmRestApi::Actions.access_filter = DppmRestApi.access_filter
+module DppmRestApi
+  # Disable all authentication checks by making `DppmRestApi::Actions.has_access?`
+  # always return true, then set the authentication back to the default system
+  # after yielding to the block.
+  def self.without_authentication!
```

Comment:
The property is private. I had method next to this which called this method, so that it would be at the global namespace, but I thought you wouldn't like the extra abstraction. We could also make the property not protected.